src/services/review/ReviewRequestService.ts

```diff
@@ -125,6 +126,29 @@ export default class ReviewRequestService {
     return mappings
   }
 
+  computeCommentDataMappings = async (
+    comments: {
+      userId: string
+      message: string
+      createdAt: string
+    }[],
```

Comment:
could we make a type for this in `types/review`? possibly through reusing `CommentItem`? eg: `Pick<CommentItem, 'message' | 'createdAt'> & {userId: string}`

Reply:
Sure, added in bd98768 - I've made it a new type because the `createdAt` from github is a string!